docs/finality-provider-operation.md

```diff
@@ -221,11 +211,11 @@ Parameters:
 
 ### 4.2. Add an EOTS Key
 
-This section explains the process of setting up the private keys for the
-EOTS manager. Operators *must* create an EOTS key before starting the
+This section explains the process of creating EOTS keys using the EOTS manager.
+Operators *MUST* create an EOTS key before starting the
 EOTS daemon.
```

Comment:
Do we have an error if operators don't do that?

Reply:
Nope. We probably can remove Line 215

Reply:
If they must do that, shouldn't we have an error?

Reply:
Maybe we can add a check during eotsd initiation. This check should examine the db record for key creation and the program should exist if it find none. There's no point of starting eotsd anyway if no key is created